src/modules/clusters/cluster-list/cluster-list.html

```diff
@@ -110,12 +108,12 @@ <h5 class="pull-left counts">{{cluster.alertCount}}</h5>
                             </div>
                             <div class="list-view-pf-additional-info-item list-view-pf-additional-info-item-stacked" ng-class="{'hide-button': (cluster.managed !== 'Yes')}">
                                 <div class="bold-text">Volume Profiling</div>
-                                <h5 class="pull-left counts">{{cluster.isProfilingEnabled}}</h5>
+                                <div class="pull-left counts" ng-show="cluster.isProfilingEnabled !== 'Pending'">{{cluster.isProfilingEnabled}}</div>
+                                <div class="pull-left counts" ng-show="cluster.isProfilingEnabled === 'Pending'"><a href="" ng-click="clusterCntrl.goToTaskDetail(cluster)">{{cluster.isProfilingEnabled}}</a></div>
                             </div>
                             <div class="list-view-pf-additional-info-item cluster-text" ng-class="{'error-message': (cluster.message === 'Cluster Misconfigured'), 'warning-message': (cluster.message === 'Import Failed' || cluster.message === 'Unmanage Failed')}">
                                 <p uib-tooltip-template="'../modules/clusters/cluster-list/unmanage-cluster-tooltip.html'" tooltip-enable="clusterCntrl.isTooltipEnable(cluster.message)" tooltip-placement="auto">{{cluster.disableExpand ? "Expanding cluster" : cluster.message}}</p>
-                                <!-- <p><a ng-click="clusterCntrl.openErrorModal(cluster)" ng-show="cluster.currentStatus === 'failed'" class="details-link">View Details</a></p> -->
-                                <p><a ng-click="clusterCntrl.goToTaskDetail(cluster)" ng-show="cluster.currentStatus === 'in_progress' || cluster.currentStatus === 'failed'" class="details-link">View Details</a></p>
+                                <p><a ng-click="clusterCntrl.goToTaskDetail(cluster)" ng-show="(cluster.currentStatus === 'in_progress' && cluster.jobType !== 'EnableDisableVolumeProfiling')|| cluster.currentStatus === 'failed'" class="details-link">View Details</a></p>
```

Comment:
same as above

Reply:
@cloudbehl This can be done via function.